.github/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# syntax=docker/dockerfile:1
```

Comment:
In `1.x`, we made use of the [geth-dev-assistant](https://github.com/ChainSafe/geth-dev-assistant) repo, and I think it would be wise to continue using the scripts: [geth-automine.sh](https://github.com/ChainSafe/web3.js/blob/1.x/scripts/e2e.geth.automine.sh) and `geth-instamine.sh](https://github.com/ChainSafe/web3.js/blob/1.x/scripts/e2e.geth.instamine.sh)